spec.v1.yml

```diff
@@ -258,15 +258,15 @@ components:
       properties:
         collection:
           type: string
-          description: the Collection's name
+          description: Name of the collection
         bookNumber:
           type: string
           description: The number of the book this hadith belongs to
         chapterId:
           type: string
-          description: The id of the chapter this hadith belongs to
+          description: The ID of the chapter this hadith belongs to
         hadithNumber:
-          type: integer
+          type: string
```

Comment:
So from the DB, this is a decimal format with 2 decimal places. Are we treating it as a string consistently as far as its exposure to consumer is concerned? (i.e. in wherever it is being exposed?)

Reply:
`hadithNumber` is a varchar in the `HadithTable` and also there are values such as `100 b` so string is correct, I believe.